pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -615,10 +608,47 @@ func (r *ReconcileChaosEngine) reconcileForDelete(request reconcile.Request) (re
 
 	// TODO: write function to remove all the chaos resources
 	// For now, it just removes Pods, especially runner and monitor
-	// With the passing of engineUID in litmus, everything can be removed from here.
+	// With the passing of chaosUID in litmus, everything can be removed from here.
 
 }
 
+func (r *ReconcileChaosEngine) removeChaosResources(engine *chaosTypes.EngineInfo, request reconcile.Request) (reconcile.Result, error) {
+	optsDelete := []client.DeleteAllOfOption{
+		client.InNamespace(request.NamespacedName.Namespace),
+		client.MatchingLabels{"chaosUID": string(engine.Instance.UID)},
+		client.PropagationPolicy(metav1.DeletePropagationForeground),
+	}
+	var deleteEvent []string
+	deleteEvent = append(deleteEvent, "Unable to delete chaos resources allocated by Chaos Engine Resources: ")
+
+	var err []error
+
+	if errDeployment := r.client.DeleteAllOf(context.TODO(), &appsv1.Deployment{}, optsDelete...); errDeployment != nil {
+		err = append(err, errDeployment)
+		deleteEvent = append(deleteEvent, "Deployments, ")
+	}
+
+	if errDaemonSet := r.client.DeleteAllOf(context.TODO(), &appsv1.DaemonSet{}, optsDelete...); errDaemonSet != nil {
+		err = append(err, errDaemonSet)
+		deleteEvent = append(deleteEvent, "Daemon Sets, ")
+	}
+
+	if errJob := r.client.DeleteAllOf(context.TODO(), &batchv1.Job{}, optsDelete...); errJob != nil {
+
+		err = append(err, errJob)
+		deleteEvent = append(deleteEvent, "Jobs, ")
+	}
+
+	if errPod := r.client.DeleteAllOf(context.TODO(), &corev1.Pod{}, optsDelete...); errPod != nil {
+		err = append(err, errPod)
+		deleteEvent = append(deleteEvent, "Pods, ")
+	}
+	if err != nil {
+		r.recorder.Eventf(engine.Instance, corev1.EventTypeWarning, "Unable to delete ChaosEngine allocated resources", "Unable to delete chaos Resources : %v", strings.Join(deleteEvent, ""))
```

Comment:
Can you pls take a look to this msg, what will be the final one?
`"Unable to delete ChaosEngine allocated resources", "Unable to delete chaos Resources : %v", strings.Join(deleteEvent, "")) ` for this command